api/src/main/java/org/openmrs/module/attachments/obs/ComplexDataHelperImpl.java

```diff
@@ -15,12 +15,12 @@
 import org.springframework.stereotype.Component;
 
 @Component(AttachmentsConstants.COMPONENT_COMPLEXDATA_HELPER)
-@OpenmrsProfile(openmrsPlatformVersion = "1.10.2 - 1.12.*")
-public class ComplexDataHelper1_10 implements ComplexDataHelper {
+@OpenmrsProfile(openmrsPlatformVersion = "2.*")
```

Comment:
Don't you require 2.4 and above?

Reply:
Yes indeed. I missed updating this from the time I was doing runtime tests with 2.2 versions of openmrs and above. I have to update those in the next commit. Thanks @dkayiwa.